oaknational/Oak-Web-Application
# LESQ-4 feat(schoolpicker): highlight bold search term

## Description

list only appears when search term has 3 characters
search term is bold on list elements

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2013--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-foundation/units/surds/lessons/accuracy-of-final-answers/downloads?preselected=all
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
